Library/Styles/BaseStyles.swift

```diff
@@ -118,6 +118,12 @@ public let separatorStyle: ViewStyle = { (view: UIView) in
     |> \.accessibilityElementsHidden .~ true
 }
 
+public let separatorStyleDark: ViewStyle = { view in
```

Comment:
Do we also have `Light` style?

Reply:
We do - the one above is called `separatorStyle` which essentially the light style. @dannyalright mentioned that he uses these two styles for separators when designing so I figured it made sense to add it to our `BaseStyles`. I could also refactor & rename `separatorStyle` to `separatorLightStyle` but that felt out of the scope of this PR.